src/js/drawer.js

```diff
@@ -50,31 +51,38 @@
           options = $.extend({}, options);
           $this.data(namespace, { options: options });
 
-          var iScroll = new IScroll('.' + __.settings.class.nav, options.iscroll);
+          __.refresh.call(_this);
 
           if (options.showOverlay) {
             __.addOverlay.call(_this);
           }
 
           $('.' + __.settings.class.toggle).on('click.' + namespace, function() {
             __.toggle.call(_this);
-            return iScroll.refresh();
+            return _this.iScroll.refresh();
           });
 
           $(window).resize(function() {
             __.close.call(_this);
-            return iScroll.refresh();
+            return _this.iScroll.refresh();
           });
 
           $('.' + __.settings.class.dropdown)
             .on(__.settings.dropdownEvents.opened + ' ' + __.settings.dropdownEvents.closed, function() {
-              return iScroll.refresh();
+              return _this.iScroll.refresh();
             });
         }
 
       }); // end each
     },
 
+    refresh: function refreshIScroll() {
```

Comment:
Ok! I always use named functions because I find them easier to debug when some error occurs (following the recommendations from Kyle Simpson), but I'll remove it. :)